From 52f75c43702ab04f3426025eea80155aa27b3422 Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@ti.com>
Date: Tue, 8 Oct 2013 00:18:10 -0400
Subject: [PATCH] qtwebkit-examples: enable building examples by default

Examples are provided in sources, but the build for them is not enabled by
default. Since example binaries are packaged separately to ${PN}-examples,
having them built by default makes more sense.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Denys Dmytriyenko <denys@ti.com>
---
 qtwebkit-examples.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qtwebkit-examples.pro b/qtwebkit-examples.pro
index 3fcecf1..2da8a36 100644
--- a/qtwebkit-examples.pro
+++ b/qtwebkit-examples.pro
@@ -2,4 +2,4 @@ requires(qtHaveModule(webkit))
 
 load(qt_parts)
 
-SUBDIRS += doc
+SUBDIRS += doc examples
-- 
2.3.5

